osu.Game.Tests/Visual/Settings/TestSceneLatencyCertifierScreen.cs

```diff
@@ -24,6 +24,18 @@ public override void SetUpSteps()
             AddUntilStep("wait for load", () => latencyCertifier.IsLoaded);
         }
 
+        [Test]
+        public void TestCircleGameplay()
+        {
+            AddStep("set visual mode to circles", () => latencyCertifier.VisualMode.Value = LatencyVisualMode.CircleGameplay);
+        }
```

Comment:
Minor note here, but since latency certifier uses circle gameplay mode by default, this test scene pretty much does nothing. Maybe should be flipped the other around and have this test scene switch to "simple" mode instead? Assuming that "circle gameplay" being default mode is intentional.

Reply:
I mainly made this to test / work on the gameplay. So the alternative would be to have it have no steps.

Reply:
I see, probably better adding another test case for the "simple" mode then. Will add one.